### PR TITLE
[SL Alpha5] Change documentation structure

### PR DESCRIPTION
**Pull request by SanduDS**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes
 -

### CODE REVIEW

File: outlookmail/Module.md [hunk 1]
```diff
@@ -2,16 +2,18 @@
...
- 
-## Configuring connector
-### Prerequisites
-* Microsoft Outlook Account
-* Azure Active Directory Access
-### Obtaining tokens
+
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* create [Microsoft Outlook Account](https://outlook.live.com/owa/)
```
Comment (abeykoon) [inline outlookmail/Module.md:8]: `c` is simple, but in next point `O` is capital. No consistency.

File: outlookmail/Module.md [hunk 2]
```diff
@@ -2,16 +2,18 @@
 Ballerina connector for Microsoft Outlook mail provides access to Microsoft Outlook mail service in Microsoft Graph v1.0 via Ballerina language easily. It provides capability to perform more useful functionalities provided in Microsoft outlook mail such as sending messages, listing messages, creating drafts, mail folders, deleting and updating messages etc. 
 
 This module supports Ballerina SL Beta 2 version.
```
Comment (abeykoon) [inline outlookmail/Module.md:4]: Should we mention this?

File: outlookmail/Module.md [hunk 3]
```diff
@@ -54,6 +56,16 @@ public function main() returns error? {
     mail:Message message = check outlookClient->createMessage(draft);
     log:printInfo(message.toString());
 }
-   
+
 ``` 
-## [You can find more samples here](https://github.com/ballerina-platform/module-ballerinax-microsoft.outlook.mail/tree/main/outlookmail/samples)
\ No newline at end of file
+## Quick reference 
+* Get a message
```
Comment (abeykoon) [inline outlookmail/Module.md:62]: Receive a message?

File: outlookmail/Module.md [hunk 4]
```diff
@@ -54,6 +56,16 @@ public function main() returns error? {
...
+ ```ballerina
+   mail:Message message = check outlookClient->getMessage("<Message ID>");
+   ```
+* Send an existing draft
```
Comment (abeykoon) [inline outlookmail/Module.md:66]: What about `sending a simple email` example? Sending a draft is something advance (beyond very basic)

Comment (SanduDS) [reply outlookmail/Module.md:66]: The quick start section has the complete sample for sending a message, here it is just get the message Id of a draft and send it.

**Review by abeykoon — APPROVED** [review]

**Pull request merged**